### PR TITLE
[API-8] Add missing Living value accessors

### PR DESCRIPTION
**Pull request by ItsDoot**
Adds value accessors for the following keys to `Living`:
- `ABSORPTION`
- `ACTIVE_ITEM`
- `EYE_HEIGHT`
- `EYE_POSITION`
- `FALL_DISTANCE`
- `MAX_AIR`
- `REMAINING_AIR`
- `STUCK_ARROWS`
- `WALKING_SPEED`

### CODE REVIEW

File: src/main/java/org/spongepowered/api/entity/living/Living.java [hunk 1]
```diff
@@ -78,6 +78,22 @@
         return this.requireValue(Keys.HEAD_ROTATION).asMutable();
     }
 
+    /**
+     * {@link Keys#EYE_HEIGHT}
+     * @return The height of the eyes
+     */
+    default Value.Immutable<Double> eyeHeight() {
```
Comment (Zidane) [inline src/main/java/org/spongepowered/api/entity/living/Living.java]: Only do Value.

I now remember the discussion ages ago. Getting a Value doesn't mean setting the `value` will actually set it back on the data holder. You have to offer it back. A DataProvider in common can reject it.

Nothing should stop your Value from not being written to how you like though.

@gabizou

Comment (ItsDoot) [reply src/main/java/org/spongepowered/api/entity/living/Living.java]: Fixed

File: src/main/java/org/spongepowered/api/entity/living/Living.java [hunk 2]
```diff
@@ -47,6 +48,22 @@
...
+     */
+    default BoundedValue.Mutable<Double> absorption() {
+        return this.requireValue(Keys.ABSORPTION).asMutable();
+    }
+
+    /**
+     * {@link Keys#ACTIVE_ITEM}
+     * @return The active item, such as food being eaten
+     */
+    default Value.Mutable<ItemStackSnapshot> activeItem() {
```
Comment (Zidane) [inline src/main/java/org/spongepowered/api/entity/living/Living.java:62]: @gabizou 

Can you give some guidance on how the Values in these interfaces should be returned to consumers?

Like, should they always be Value.Mutable? Just Value? Does it depend on what it is returning and what you can do with it in the game?

Comment (gabizou) [reply src/main/java/org/spongepowered/api/entity/living/Living.java:62]: I am preferring they're always `Mutable` from entities. It does make sense that they are reflective of their `Key` variables, but at the end of the day, the developer is likely to want to mutate the value and maybe offer it back, or present it to another entity after mutation.

**Pull request merged**